antora-playbook-snippets/antora-playbook.yml

```diff
@@ -60,20 +59,13 @@ content:
         - main
         - 0.10.x
         - 0.9.x
-        - 0.7.x
       start_path: docs
 
     - url: https://github.com/apache/camel-quarkus.git
       branches:
         - main
         - 2.14.x
```

Comment:
@davsclaus We could add 2.15.x, as I forgot to do so.

Reply:
Yes I am adding it